### PR TITLE
Aaron/create movies

### PR DESCRIPTION
**Pull request by ZacharyEllixson**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -27,6 +27,11 @@ def create_movies_form():
 @app.post('/movies')
 def create_movie():
     # TODO: Feature 2
+    movie_name = request.form['movieName']
+    director_name = request.form['directorName']
+    rating = request.form['selectRating']
```
Comment (joselara22) [inline app.py:37]: Have you considered using request.form.get() so an exception isn't thrown if the name is ever not found?

**Review by joselara22 — CHANGES_REQUESTED** [review]
Everything looks good.

File: app.py [hunk 2]
```diff
@@ -27,6 +27,11 @@ def create_movies_form():
...
+    get_movie_repository().create_movie(movie_name, director_name, int(rating))
+    print(movie_name, director_name, int(rating))
```
Comment (WKirklandJr) [inline app.py:39]: Good idea printing it out to verify that the code is working

File: templates/create_movies_form.html
Comment (WKirklandJr) [inline templates/create_movies_form.html:1]: Is there a reason for the aria-desribedby attribute? Were you planning to reference it later?

**Review by WKirklandJr — COMMENTED** [review]
Looks good just confirm with others

File: templates/create_movies_form.html
Comment (alobo2) [reply templates/create_movies_form.html:1]: Ooh I was using my previous html code and totally forgot to take out that attribute and id, nice catch👍

I've now removed it.

**Pull request merged**